src/app/public-view/components/projects-gallery/components/project-gallery/project-gallery.component.ts

```diff
@@ -25,8 +26,13 @@ export class ProjectGalleryComponent implements OnInit {
   }
 
   ngOnInit() {
+    this.loading = true;
     this.fileService.getFileFromPath(this.project.titleImage).subscribe(photo => {
-      this.image = this.sanitizer.bypassSecurityTrustUrl(URL.createObjectURL(new Blob([photo], {type: 'application/octet-stream'})))
+      this.image = this.sanitizer.bypassSecurityTrustUrl(
+        URL.createObjectURL(new Blob([photo],
+          {type: 'application/octet-stream'}))
+      );
+      this.loading = false;
```

Comment:
ak tu jebne error, `this.loading = false` sa asi nevykona a tym padom spinner zostane

Reply:
Ked nepojde nacitat obrazok tak mozno by to tak bolo fajn nechat lebo nemas potom co ine zobrazit :D

Reply:
okay, moze byt aj tak, ja som mal skor na mysli nejaky vypis ze nepodarilo sa nacitat, skuste znovu... ale moze byt aj spinner